client/blocks/comments/post-comment.jsx

```diff
@@ -168,10 +168,20 @@ class PostComment extends Component {
 						: <Gravatar user={ comment.author } /> }
 
 					{ authorUrl
-						? <a href={ authorUrl } className="comments__comment-username" onClick={ this.handleAuthorClick }>
+						? <a
+							href={ authorUrl }
+							className="comments__comment-username"
+							onClick={ this.handleAuthorClick }
+							name={ `comment-${ this.props.commentId }` }
```

Comment:
probably better to use `id` here rather than `name`. https://stackoverflow.com/a/484781